docs/Unity-Inference-Engine.md

```diff
@@ -32,4 +32,6 @@ Uncheck the `Control` checkbox for the corresponding **Brain** in the
 **BroadcastHub** of the Academy.
 Select the **Inference Device** : CPU or GPU you want to use for Inference.
 
-_Note: For most of the models generated with the ML-Agents toolkit, CPU will be faster than GPU. Only use GPU if you have a large number of agents using visual observations._
+**Note:** For most of the models generated with the ML-Agents toolkit, CPU will be faster than GPU.
+You should use the GPU only if you have a large number of agents using visual observations or use the 
```

Comment:
`...observations, or...`? It's ambiguous if you have a small number of agents with ResNet.